Sources/SwiftSyntax/Identifier.swift

```diff
@@ -12,23 +12,38 @@
 
 /// A canonicalized representation of an identifier that strips away backticks.
 public struct Identifier: Equatable, Hashable, Sendable {
-  /// The sanitized `text` of a token.
+  /// The sanitized name of the identifier.
   public var name: String {
     String(syntaxText: raw.name)
   }
 
   @_spi(RawSyntax)
   public let raw: RawIdentifier
-
-  private let arena: SyntaxArenaRef
+  let arena: SyntaxArenaRef?
```

Comment:
Why do we need to make `arena` non-private? It should only be used to keep the arena alive and we shouldn’t expose it.